apps/demo/src/scss/mixins/_dividers.scss

```diff
@@ -1,3 +1,5 @@
+@import 'daff-theme';
```

Comment:
I think this should be refactored into a `divider` component rather than a mixin so it can be used in broader instances.

Reply:
I agree we shouldn't do it in this PR, but there are many instances where I'd rather use a `divider` component than having to set border-radius on a specific container (that I might have override later).

Reply:
I thought the divider was the line that sometimes separates sections of the page. What does that have to do with border-radius or other components?

Reply:
Oops, I mean border-top/border-bottom. I talked to Damien about this and it's not in the scope of this PR so I'm going to mark it as resolved.